packages/assets-controllers/src/TokenRatesController.ts

```diff
@@ -217,20 +224,49 @@ export class TokenRatesController extends BaseController<
       this.update({ contractExchangeRates: {} });
       this.configure({ chainId, nativeCurrency: ticker });
     });
-    this.poll();
   }
 
   /**
-   * Sets a new polling interval.
+   * Start (or restart) polling.
    *
    * @param interval - Polling interval used to fetch new token rates.
    */
-  async poll(interval?: number): Promise<void> {
-    interval && this.configure({ interval }, false, false);
-    this.handle && clearTimeout(this.handle);
+  async start(interval?: number) {
```

Comment:
Since this is a new method, what do you think about having this method take an options bag, with `interval` as one of the options, instead of a positional argument? Just thinking that this would make it easier to understand what the number represents at a glance.

Reply:
Sure, good idea.

I was tempted to remove it as well - do you think that might be preferable? I don't think we make use of this feature anywhere.

Reply:
Removed in https://github.com/MetaMask/core/pull/1501/commits/1a468bf0351efaf8c8a2603ce20c725cb0acdd19